tmk_core/common/keycode.h

```diff
@@ -481,9 +484,9 @@ enum internal_special_keycodes {
     KC_MS_WH_LEFT,
     KC_MS_WH_RIGHT,     /* 0xFC */
     /* Mousekey accel */
-    KC_MS_ACCEL0,
-    KC_MS_ACCEL1,
-    KC_MS_ACCEL2        /* 0xFF */
+    KC_MS_ACCEL3,
+    KC_MS_DNSPED,
+    KC_MS_UPSPED        /* 0xFF */
```

Comment:
Had to remove the old ACCEL to be able to have enough room to add the new modifiers, might be best to remove some of the FN keys higher up in this file